web/pgadmin/browser/server_groups/servers/databases/publications/__init__.py

```diff
@@ -691,6 +821,34 @@ def get_sql(self, data, pbid=None):
                                   data=data, conn=self.conn)
             return sql.strip('\n'), data['name']
 
+    @login_required
+    def get_schemas(self, gid, sid, did):
+        """
+        This function will return the list of schemas for the specified
+        server group id, server id and database id.
+
+        Args:
+            gid: Server Group ID
+            sid: Server ID
+            did: Database ID
+        """
+        res = []
+        try:
+            view = SchemaDiffRegistry.get_node_view('schema')
```

Comment:
we should not use `SchemaDiffRegistry` here. It is meant for using in SchemaDiff only.